src/js.rs

```diff
@@ -612,6 +612,82 @@ extern {
     pub fn value_of(this: &Object) -> Object;
 }
 
+// Set
+#[wasm_bindgen]
+extern {
+    pub type Set;
+
+    /// The add() method appends a new element with a specified value to the end of a Set object.
+    /// 
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add
+    
```

Comment:
Minor nitpick, It would be great if you could remove this line so that the comment is just above. `#[wasm_bindgen]`